scripts/update-anserini.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+
+CORD19_INDEX_NAME=lucene-index-cord19-paragraph
+CORD19_DATE=$1
+COVIDEX_CORD19_INDEX_PATH=api/index/${CORD19_INDEX_NAME}
+
+TRIALSTREAMER_INDEX_NAME=lucene-index-trialstreamer
+TRIALSTREAMER_INDEX_DATE=2020-04-15
+TRIALSTREAMER_INDEX_URL=https://www.dropbox.com/s/d2s92i6y927s1c7/lucene-index-trialstreamer-2020-04-15.tar.gz
+
+
+# Check if date provided
+if [ $# -eq 0 ]; then
+    echo "USAGE: sh scripts/update-anserini.sh [DATE]"
+    exit 1
+fi
+
+
+echo "Updating CORD-19 index for $1..."
+[ -d anserini ] && echo "Found Anserini folder..." || git clone https://github.com/castorini/anserini.git
+python3 -m pip install tqdm
```

Comment:
`tqdm` should be in the requirements already, right?

Reply:
It's not, but I'll add it there